Userland/Libraries/LibJS/Bytecode/Generator.cpp

```diff
@@ -426,7 +426,8 @@ void Generator::generate_scoped_jump(JumpType type)
             break;
         case ReturnToFinally: {
             auto jump_type_name = type == JumpType::Break ? "break"sv : "continue"sv;
-            auto& block = make_block(DeprecatedString::formatted("{}.{}", current_block().name(), jump_type_name));
+            auto block_name = String::formatted("{}.{}", current_block().name(), jump_type_name).release_value_but_fixme_should_propagate_errors();
```

Comment:
This looks good, but these `release_value_but_fixme_should_propagate_errors` can just be a MUST() around the expression.

Reply:
Done. Thanks for the tip